pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -153,10 +154,11 @@ func (l *libcontainerAdapter) newManager(cgroups *libcontainerconfigs.Cgroup, pa
 		if !cgroupsystemd.UseSystemd() {
 			panic("systemd cgroup manager not available")
 		}
-		return &cgroupsystemd.LegacyManager{
-			Cgroups: cgroups,
-			Paths:   paths,
-		}, nil
+		f, err := cgroupsystemd.NewSystemdCgroupsManager()
```

Comment:
does this manager work on v1 and v2 hosts?

Reply:
i need to check the runc code, but is this abstracing legacy from unified manager?

Reply:
yes, `NewSystemdCgroupsManager()` checks for the cgroup version used:

https://github.com/opencontainers/runc/blob/master/libcontainer/cgroups/systemd/apply_systemd.go#L118-L136

If we are running on cgroup v1, it will use `cgroupsystemd.LegacyManager`

Reply:
thanks for pointer, and noting that i confirmed this is in the vendored version.